### DISCUSSION TITLE
[hexo主题]能否每偏博文的banner也做一个开关

### DISCUSSION THREAD

**Discussion by Angel0726** [tangly1024/NotionNext#2038] [Ideas]
https://github.com/tangly1024/NotionNext/blob/main/themes/hexo/config_hexo.js

**为什么提出这个新的特性改动**
简要说明此特性解决的问题，例如，『博客站点的读者互动性不够强，和读者无法建立紧密的联系...』

**描述一下你推荐的解决方案**
简要说明你的解决方案建议，例如，『Giscus评论插件功能更加强大，用户只需留言既可在你的邮箱收到通知。。。』

**描述一下你考虑过的其它替代解决方案**
简要说明你所有想过的有可能解决此问题的方案。

**补充说明**
补充与此特性相关的内容

**Comment by tangly1024** [comment]
您是希望文章的内容页也支持关闭banner吗

**Comment by Angel0726** [comment]
> 

对，感觉动画有一些卡顿
